mscore/preferences.cpp

```diff
@@ -1154,13 +1154,13 @@ void PreferenceDialog::resetShortcutClicked()
     
 void PreferenceDialog::saveShortcutListClicked()
       {
-      QString saveFileName = QFileDialog::getSaveFileName(this, tr("Save Shortcuts"), "", tr("MuseScore Shortcuts File") + " (*.xml)");
+      QString saveFileName = QFileDialog::getSaveFileName(this, tr("Save Shortcuts"), "", tr("MuseScore Shortcuts File") + " (*.xml)", 0, preferences.nativeDialogs ? QFileDialog::Options() : QFileDialog::DontUseNativeDialog);
```

Comment:
Very good. I'd still like to know why it has been done in that much more complicated and convoluted way in other places, if it can be done as simple as you do it here, @lasconic?